### ISSUE TITLE
preactjs-signals style API?

### ISSUE THREAD

**Issue by FlatMapIO** [vobyjs/voby#18]
voby is perfect for embedding in my application, thank you for making this!

`signal() / atom()` and `atom.value` style getters and setters seem easier to understand, I hope to consider this.

**Comment by fabiospampinato** [comment]
Thanks for the suggestion, but I think our API for signals is better.

I wrote a little [here](https://twitter.com/fabiospampinato/status/1574041996607594496) regarding why I don't like Preact's API. Also I forgot to mention there that it's easier to shoot yourself in the foot because you need to remember to untrack manually way more often, while in Voby you can do like `count ( prev => prev + 1 )`, which is implicitly untracked.

If you are also suggesting getter/setter segregation I touched on that as part of [this](https://github.com/vobyjs/oby/issues/2) other issue.

Basically I think I've given the matter a fair amount of thought, and our current API for signals seems the best to me.
